### PR TITLE
1810 Anten Skrabec

### PR DESCRIPTION
**Pull request by cebarks**

### CODE REVIEW

File: lib/house.rb [hunk 1]
```diff
@@ -0,0 +1,48 @@
+require './lib/room'
```
Comment (memcmahon) [inline lib/house.rb]: You only need to require a class file if you are calling class methods on that class in this file (Room.method_name)

File: lib/house.rb [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+    @rooms.select do |room|
+      room.catagory == catagory
+    end
+  end
+
+  def price_per_square_foot
+    (@price / area.to_f).round(2)
+  end
+
+  def rooms_sorted_by_area
```
Comment (memcmahon) [inline lib/house.rb]: Keep a close eye on the spec - we want rooms sorted in _descending_ order; from largest to smallest area.

File: lib/house.rb [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+    @rooms.sort_by do |room|
+      room.area
+    end
+  end
+
+  def rooms_by_category
```
Comment (memcmahon) [inline lib/house.rb]: also look into the `group_by` method :)

File: lib/room.rb [hunk 4]
```diff
@@ -0,0 +1,12 @@
+class Room
+  attr_reader :x, :y, :catagory
+  def initialize(catagory, x, y)
+    @catagory = catagory
+    @x = x
```
Comment (memcmahon) [inline lib/room.rb]: Try to give things more descriptive names - like `@length` and `@width` for example

File: test/house_test.rb [hunk 5]
```diff
@@ -0,0 +1,75 @@
...
+    add_rooms
+    expected = {
+      :bedroom => [@room_1, @room_2],
+      :living_room => [@room_3],
+      :basement => [@room_4]
+    }
+    assert_equal expected, @house.rooms_by_category
+  end
+
+  def add_rooms
```
Comment (memcmahon) [inline test/house_test.rb]: I would prefer to see this helper method up near your `setup` method

File: lib/house.rb (hunk 1 above)
Comment (cebarks) [reply lib/house.rb]: Oh gotcha! That makes sense.

File: lib/house.rb (hunk 3 above)
Comment (cebarks) [reply lib/house.rb]: Yup, learned about this right after I finished >.<

**Pull request closed without merging**